man/man1/git-town-prepend.1

```diff
@@ -0,0 +1,44 @@
+.TH "GIT-TOWN-PREPEND" "1" "12/26/2016" "Git Town 2\&.1\&.0" "Git Town Manual"
+
+.SH "NAME"
+git town-prepend \- create a new feature branch between the current branch and its parent
+
+
+.SH "SYNOPSIS"
+\fIgit town-prepend\fR <branch_name>
+.br
+\fIgit town-prepend\fR (--abort | --continue)
+
+
+.SH "DESCRIPTION"
+
+.PP
+Syncs the given parent branch (default: main branch),
+.br
+forks a new feature branch with the given name off its parent,
+.br
+makes the new branch the parent of the current branch,
+.br
+pushes the new feature branch to the remote repository,
+.br
+and brings over all uncommitted changes to the new feature branch.
+
+.PP
+Additionally, when there is a remote upstream,
+.br
+the main branch is synced with its upstream counterpart.
+.br
+This can be disabled by toggling the "hack-push-flag" configuration:
+.br
+$ git town hack-push-flag false
+
+
+.SH "OPTIONS"
+.IP "<branch_name>" 4
+The name of the branch to create.
+
+.IP "--abort" 4
+Cancel the operation and reset the workspace to a consistent state.
+
+.IP "--continue" 4
+Continue the operation after resolving conflicts.
```

Comment:
Not sure this will end up simpler. This would save us 2 lines of documentation, but send readers on a journey across several man pages, which is cumbersome. Also, since I see the new `git hack` playing a more peripheral role moving forward (it's more an edge case, not more central than `git append`). I have added a "see also" section to the documentation to link to the other commands, to make clear how they are related.